doc/introduction/remote.rst

```diff
@@ -443,68 +428,15 @@ Rgate(-0.8273) | (q[0])
 Rgate(-2.51) | (q[5])
 MeasureFock | (q[0], q[1], q[2], q[3], q[4], q[5], q[6], q[7])
 
-The squeezing values required to embed this bipartite graph are given by
-the following relation:
-
->>> from thewalrus.quantum import find_scaling_adjacency_matrix
->>> c = find_scaling_adjacency_matrix(A, 2*4*0.345274461385554870545)
->>> set(np.arctanh(np.linalg.svd(c*A)[1]))
-{0.0, 1.0000000000000002}
-
-Note that the above squeezing values must be of the form :math:`\{0,1\}`
-to be embedded on the chip. Consider a bipartite graph where this is not
-the case:
+If the bipartite graph to be embedded does not satisfy the aforementioned
+restriction on the singular values, an error message will be raised on
+compilation:
 
 >>> B = np.array([[0, 1, 0, 1], [1, 0, 1, 0], [0, 1, 1, 1], [1, 0, 1, 0]])
 >>> A = np.block([[0*B, B], [B.T, 0*B]])
->>> c = find_scaling_adjacency_matrix(A, 2*4*1)
->>> set(np.arctanh(np.linalg.svd(c*A)[1]))
-{0.0,
-3.2937343775007984e-32,
-0.17674864137317442,
-0.17674864137317453,
-0.8180954232791708,
-0.8180954232791715,
-1.3361892276414615}
-
-The program will fail to compile for chip2:
-
-.. code-block:: python3
-
-    prog = sf.Program(8)
-
-    with prog.context as q:
-        ops.BipartiteGraphEmbed(A, mean_photon_per_mode=1) | q
-        ops.MeasureFock() | q
-
-    prog.compile("chip2").print()
-
-.. code-block:: bash
-
-    ---------------------------------------------------------------------------
-
-    CircuitError                              Traceback (most recent call last)
-
-    <ipython-input-23-f713320d8c3b> in <module>
-          5     ops.MeasureFock() | q
-          6
-    ----> 7 prog.compile("chip2").print()
-
-
-    ~/Dropbox/Work/Xanadu/sf_cloud/strawberryfields/program.py in compile(self, target, **kwargs)
-        522         # does the circuit spec  have its own compilation method?
-        523         if db.compile is not None:
-    --> 524             seq = db.compile(seq, self.register)
-        525
-        526         # create the compiled Program
-
-
-    ~/Dropbox/Work/Xanadu/sf_cloud/strawberryfields/circuitspecs/chip2.py in compile(self, seq, registers)
-        137             raise CircuitError(
-        138                 "Incorrect squeezing value(s) (r, phi)={}. Allowed squeezing "
-    --> 139                 "value(s) are (r, phi)={}.".format(wrong_params, allowed_sq_value)
-        140             )
-        141
-
-
-    CircuitError: Incorrect squeezing value(s) (r, phi)={(1.336, 0.0), (0.177, 0.0), (0.818, 0.0)}. Allowed squeezing value(s) are (r, phi)={(1, 0.0), (0.0, 0.0)}.
+>>> prog = sf.Program(8)
+>>> with prog.context as q:
+...     ops.BipartiteGraphEmbed(A, mean_photon_per_mode=1) | q
+...     ops.MeasureFock() | q
+CircuitError: Incorrect squeezing value(s) (r, phi)={(1.336, 0.0), (0.177, 0.0), (0.818, 0.0)}.
+Allowed squeezing value(s) are (r, phi)={(1, 0.0), (0.0, 0.0)}.
```

Comment:
I'm not sure if this is that useful for the tutorial. If a user did run into the error message, it is quite self explanatory.